hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -28,7 +28,8 @@
                         "^UVM_WARNING\\s[^:].*$",
                         "^Assert failed: ",
                         "^\\s*Offending '.*'",
-                        "^TEST FAILED (UVM_)?CHECKS$"]
+                        "^TEST FAILED (UVM_)?CHECKS$",
+                        "^Error:.*$"]  // ISS errors
```

Comment:
Sorry, I could be wrong, but I vaguely recall seeing the ISS error message in all caps `ERROR: something`.

Reply:
I think Sri's right here. Run `git grep -i error: hw/ip/otbn` to see the various messages that can come out.

Reply:
Oops, my apology. Thank you for catching that, yes I believe Sri was right as well. The error is not all caps..

Reply:
Hmm, my mistake. I intended to write "^ERROR:". The CC files use that pattern. But the python files probably need r"\w*Error:.*", as in "RuntimeError: bla blue".

Reply:
By the way, the ".*$" part in some of these regex's seems perfectly redundant.